docs/configuration/config-file-reference.md

```diff
@@ -1064,7 +1164,7 @@ storage:
 
   # The configstore_config configures the config database storing rules and
   # alerts, and is used by the Cortex alertmanager.
-  # The CLI flags prefix for this block config is: alertmanager
+  # The CLI flags prefix for this block config is: alertmanager.client
```

Comment:
This is a config breaking change we can't affort. Alertmanager is not among the list of experimental features.